miner/worker.go

```diff
@@ -872,17 +872,23 @@ func (w *worker) commitTransactions(txs *types.TransactionsByPriceAndNonce, coin
 		return true
 	}
 
+	var reservedGas uint64 = 0
+	if w.current.header.Number.Uint64()%w.chainConfig.Consortium.EpochV2 == w.chainConfig.Consortium.EpochV2-1 {
```

Comment:
We need to add check for `pool.chainconfig.Consortium != nil` to avoid crash on unit test